lib/pure/strutils.nim

```diff
@@ -444,6 +444,10 @@ proc strip*(s: string, leading = true, trailing = true,
   ##
   ## If `leading` is true, leading `chars` are stripped.
   ## If `trailing` is true, trailing `chars` are stripped.
+  ##
+  ## `chars` are only stripped when those are at the beginning
+  ## or ending of `s`. Use `replace <#replace,string,string>`_
```

Comment:
Does this link really work? How can replace for strings be used to strip a set of characters?

Reply:
`./koch web` fails (also without my changes), so I don't really know if the link works:
```
algorithm.nim(428, 51) Error: undeclared identifier: 'HSlice'
[Examples] failed
```

Did you read #7159 ? If you have a better suggestion for my problem, I'm interested.